src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -124,15 +138,39 @@ const MapOverlays = ({
               }
               return poi.level === indoorInformation.currentFloor.level;
             })
-            .map(poi => (
-              <CustomMarker
-                markerType="poi"
-                key={poi.id}
-                location={poi.location}
-                text={poi.displayName}
-                onPress={() => {}}
-              />
-            ))}
+            .map(poi => {
+              if (
+                startTravelPlan &&
+                ((destination && destination.id === poi.id) ||
+                  (initialLocation && initialLocation.id === poi.id))
+              ) {
+                return (
+                  <CustomMarker
+                    markerType="poi"
+                    key={poi.id}
+                    location={poi.location}
+                    text={poi.displayName}
+                    onPress={() => {
+                      setMarkerLocation(poi);
+                    }}
+                  />
+                );
+              }
+              if (!startTravelPlan) {
+                return (
+                  <CustomMarker
+                    markerType="poi"
+                    key={poi.id}
+                    location={poi.location}
+                    text={poi.displayName}
+                    onPress={() => {
+                      setMarkerLocation(poi);
+                    }}
+                  />
+                );
+              }
+              return null;
+            })}
```

Comment:
Add another blow which display those two marker only if there is currently a travel plan